### DISCUSSION TITLE
Is ToonSquid still a part time job? (Question to Dev)

### DISCUSSION THREAD

**Discussion by Lioncub901** [keiwando/toonsquid#667] [Q&A]
I have noticed that in the app store that the ToonSquid app has be getting quite a lot or people downloading the app. I just want to know. Is ToonSquid profitable enough to become a full time job or is it still a part time job? 

Edit: Oh, I am not saying that he should make it into a full time job. I just saying would the income coming from there be enough to be a full time job. Just curious.

**Comment by Rohitgit2412** [comment]
Keiwan, the developer, has a full-time job. In addition to that, he has also developed two other software programs: Evolution and other's name I'm forgetting. He manages both of these software programs alongside his full-time job and ToonSquid.

**Comment by FoxeyeStudios** [comment]
ToonSquid isn’t going to be a full time job, it’s a passion shared to the world. Keiwan not only made ToonSquid very affordable, he also only made it as a one-time purchase. Meaning, even though it’s currently getting more revenue, it’s still not enough to live off of since every consumer only needs to pay once and then own it indefinitely. It could be said that all the purchases can be a big help to him, but he still needs a more stable source of income. That’s why he already has a full-time job, it’s uncertain whenever he’s going to receive money from app purchases since I imagine they come at random.

**Comment by brindacier** [comment]
Many may wish to pay for the 2.0 ! At least me
